### ISSUE TITLE
Remove `node_modules/` from type checks

### ISSUE THREAD

**Issue by joshestein** [bluedotimpact/bluedot#2112]
### 🧩 Summary

react-aria-components have some failing type checks which leads to problems running `tsc`

### ✅ Definition of done

`node_modules` aren't included in type checks, `npx next build` works locally
